ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-create-run-utils.ts

```diff
@@ -161,6 +162,49 @@ const convertArrayToMap = (
   );
 };
 
+// Define the type for the returned error messages with full paths
+type FlattenedError = {
+  path: string;
+  message: string;
+};
+
+/**
+ * Flatten the error object returned by react-hook-form to an array of error messages with full paths.
+ *
+ * @param errors - An array of error objects to be processed.
+ * @param path - The path to the current object in the object tree.
```

Comment:
Can you add some documentation on what the path does as it's not immediately apparent from the code?